docs/guide/search/search_synonyms.md

```diff
@@ -16,24 +30,18 @@ With this method you create a managed synonym resource that can be updated via R
 </fieldType>
 ```
 
-This definition should be placed in `schema.xml` file. It defines `solr.ManagedSynonymFilterFactory` for synonym main class instead of `SynonymFilterFactory`.
-
-Now you have to add the dynamic field definitions for the fields that you want to use as synonyms.
+This definition should be placed in the `schema.xml` file. It defines `solr.ManagedSynonymFilterFactory` for synonym main class instead of `SynonymFilterFactory`.
```

Comment:
```suggestion
You must place this definition in the `schema.xml` file. 
It defines `solr.ManagedSynonymFilterFactory` for synonym main class instead of `SynonymFilterFactory`.
```